packages/ckeditor5-package-generator/package.json

```diff
@@ -7,7 +7,7 @@
   "homepage": "https://ckeditor.com/ckeditor-5",
   "main": "lib/index.js",
   "engines": {
-    "node": ">=22.0.0"
+    "node": ">=24.11.0"
```

Comment:
We should probably add it to `packages/ckeditor5-package-tools/package.json` as well.